packages/mobile/src/send/utils.ts

```diff
@@ -171,21 +180,44 @@ export function showLimitReachedError(
   return showError(ErrorMessages.PAYMENT_LIMIT_REACHED, ALERT_BANNER_DURATION, translationParams)
 }
 
+// exported for testing
+export function* _updateRecipientCache(recipient: RecipientWithQrCode) {
```

Comment:
Please correct me if I am wrong but I don't think we use the underscore convention as part of the wallet code base. I understand you want to show that this is a private helper function but I don't want it to be confusing because we aren't doing it anywhere else